src/filter-image/FilterImage.tsx

```diff
@@ -0,0 +1,61 @@
+import { ImageProps, View, Image as RNImage, StyleSheet } from 'react-native';
+import {
+  Svg,
+  FeColorMatrix,
+  Filter,
+  Image,
+  FeColorMatrixProps,
+} from '../index';
+import { FilterPrimitiveStandardAttributes } from '../elements/filters/FilterPrimitive';
+
+export interface FilterImageProps extends ImageProps {
+  filters: Array<
+    ({ name: 'colorMatrix' } & FeColorMatrixProps) &
+      FilterPrimitiveStandardAttributes
+  >;
+}
+
+const getFilters = (filters: FilterImageProps['filters']) => {
+  return filters?.map((filter, index) => {
+    const { name, ...filterProps } = filter;
+    switch (name) {
+      case 'colorMatrix':
+        return <FeColorMatrix key={`${filter}-${index}`} {...filterProps} />;
+      default:
+        return null;
+    }
+  });
+};
+
+export const FilterImage = (props: FilterImageProps) => {
+  const { source, style, ...imageProps } = props;
+
+  const src = RNImage.resolveAssetSource(source);
+  const styles = StyleSheet.flatten(style);
```

Comment:
If `style` comes from `ImageProps`, why are we passing it to `View`? Based on https://reactnative.dev/docs/image-style-props#props, they seem to differ. I know it should be a wrapper around the `Image` component, but maybe we should introduce some new prop to style the `View` that surrounds it? Also, doesn't `Svg` take all the props that are available on `View`? Why do we need another wrapping `View`?

Reply:
I believe it should not produce any errors because we handle the Image's special properties, like `resizeMode`, separately.